From 4f8ffc48a5abfa34f6e3a166de591ca39e596f63 Mon Sep 17 00:00:00 2001
From: Jayanthi Annadurai <jannadurai@marvell.com>
Date: Tue, 15 Jun 2021 14:23:26 -0700
Subject: [PATCH 1587/1921] drivers: marvell: rpram: Change rpram driver name

RPRAM driver is applicable for CN10K platform.

Change-Id: If878b48459648b3bdf6ddbb14fcfc0cb901683af
Signed-off-by: Jayanthi Annadurai <jannadurai@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/54166
Reviewed-by: Sunil Kovvuri Goutham <sgoutham@marvell.com>
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Tested-by: Sunil Kovvuri Goutham <sgoutham@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/soc/marvell/Makefile                            | 2 +-
 drivers/soc/marvell/{octeontx-rpram.c => cn10k-rpram.c} | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename drivers/soc/marvell/{octeontx-rpram.c => cn10k-rpram.c} (100%)

diff --git a/drivers/soc/marvell/Makefile b/drivers/soc/marvell/Makefile
index 633f733f5ce4..754f4b0563f9 100644
--- a/drivers/soc/marvell/Makefile
+++ b/drivers/soc/marvell/Makefile
@@ -12,4 +12,4 @@ obj-$(CONFIG_OCTEONTX_INFO) += octeontx_info.o
 obj-$(CONFIG_MVMDIO_UIO) += mvmdio_uio.o
 obj-$(CONFIG_MRVL_PHY_DIAGNOSTICS) += phy_diag.o
 obj-$(CONFIG_MARVELL_CN10K_MAC_MGMT) += marvell_mac_mgmt.o
-obj-$(CONFIG_MARVELL_CN10K_RPRAM) += octeontx-rpram.o
+obj-$(CONFIG_MARVELL_CN10K_RPRAM) += cn10k-rpram.o
diff --git a/drivers/soc/marvell/octeontx-rpram.c b/drivers/soc/marvell/cn10k-rpram.c
similarity index 100%
rename from drivers/soc/marvell/octeontx-rpram.c
rename to drivers/soc/marvell/cn10k-rpram.c
-- 
2.31.1

